src/main/java/de/retest/surili/commons/ga/TestLengthFitness.java

```diff
@@ -0,0 +1,21 @@
+package de.retest.surili.commons.ga;
+
+import de.retest.surili.commons.core.TestSuite;
+import lombok.NonNull;
+import lombok.Value;
+
+/**
+ * Length of all test cases of one test suite. Smaller tests produce a smaller fitness value which is better.
```

Comment:
AFAIK fitness is always maximized, but objectives can be both maximized and minimized. So we could either introduce something like an `Objective` interface and/or one has to do:

```java
@Override
public double getFitness() {
	// Multiply by -1 to enable minimization.
	return -1 * /* actual calcualtion */ ;
}
```

?